prive/tests/test_dataset.py

```diff
@@ -5,10 +5,102 @@
 
 filterwarnings('ignore')
 
-from prive.datasets.dataset import TabularDataset
+from prive.datasets import TabularDataset
 
 
 class TestTabularDataset(TestCase):
```

Comment:
There's a lot of repeated code in each of these methods. You can define a `setUp` method, as done [here](https://github.com/alan-turing-institute/synthetic_data_release/blob/master/tests/test_attacks.py), and, e.g. set `self.data = TabularDataset.read('tests/data/texas')`.

Reply:
great idea, done!